docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -101,3 +172,160 @@ The name of an alert is always the same as the job ID of the associated
 {anomaly-job} that triggered it. You can mute the notifications for a particular 
 {anomaly-job} on the page of the rule that lists the individual alerts. You can 
 open it via *{alerts-ui}* by selecting the rule name.
+
+
+[[action-variables]]
+== Action variables
+
+You can add different variables to your action. The following variables are 
+specific to the {ml} rule types.
+
+
+[[anomaly-alert-action-variables]]
+=== {anomaly-detect-cap} alert action variables
+
+Every {anomaly-detect} alert has the following action variables:
+
+`context`.`anomalyExplorerUrl`::
+URL to open in the Anomaly Explorer.
+
+`context`.`isInterim`::
+Indicates if top hits contain interim results.
+
+`context`.`jobIds`::
+List of job IDs that triggered the alert.
+
+`context`.`message`::
+A preconstructed message for the alert.
+
+`context`.`score`::
+Anomaly score at the time of the notification action.
+
+`context`.`timestamp`::
+The bucket timestamp of the anomaly.
+
+`context`.`timestampIso8601`::
+The bucket timestamp of the anomaly in ISO8601 format.
+
+`context`.`topInfluencers`::
+The list of top influencers.
+
+`context`.`topRecords`::
+The list of top records.
```

Comment:
I wonder if we should include properties of the influencers and records 🤔

Reply:
@darnautov Can you provide a list of properties?